src/Relation/Binary/PropositionalEquality.agda

```diff
@@ -20,7 +20,7 @@ open import Relation.Binary.Definitions using (DecidableEquality)
 open import Relation.Binary.Indexed.Heterogeneous
   using (IndexedSetoid)
 import Relation.Binary.Indexed.Heterogeneous.Construct.Trivial
-  as Trivial
+  as Trivial using (indexedSetoid)
```

Comment:
I thought that we had standardised on having the qualified name on the same line, and `using` on the subsequent?
ie
```agda
import Relation.Binary.Indexed.Heterogeneous.Construct.Trivial as Trivial
  using (indexedSetoid)
```
but maybe if lines get too long, then have linebreak preferred to run-on, ie.
```agda
import Relation.Binary.Indexed.Heterogeneous.Construct.Trivial
  as Trivial
  using (indexedSetoid)
```
so that each qualifying directive stands on a separate line?

Reply:
I'll give way on this one.